packages/sources/xbto/src/endpoint/price.ts

```diff
@@ -22,9 +22,12 @@ export const execute: ExecuteWithConfig<Config> = async (input, _, config) => {
 
   const jobRunID = validator.validated.id
   const market = validator.validated.data.market
-  const url = `https://fpiw7f0axc.execute-api.us-east-1.amazonaws.com/${
-    endpoints[market.toLowerCase()]
-  }`
+  const url = util.buildUrl(
+    'https://fpiw7f0axc.execute-api.us-east-1.amazonaws.com',
```

Comment:
Can we move this to the axios config / the EA config, like we do with the other EAs?

Reply:
Do you mean to set `https://fpiw7f0axc.execute-api.us-east-1.amazonaws.com` as the base URL, rather than doing it manually in the endpoint?

Reply:
Yes